windows-versions.js

```diff
@@ -24,5 +24,6 @@ export const versions = {
   "2.6.4": "https://github.com/oneclick/rubyinstaller2/releases/download/RubyInstaller-2.6.4-1/rubyinstaller-2.6.4-1-x64.7z",
   "2.6.5": "https://github.com/oneclick/rubyinstaller2/releases/download/RubyInstaller-2.6.5-1/rubyinstaller-2.6.5-1-x64.7z",
   "2.7.0": "https://github.com/oneclick/rubyinstaller2/releases/download/RubyInstaller-2.7.0-1/rubyinstaller-2.7.0-1-x64.7z",
-  "head": "https://github.com/oneclick/rubyinstaller2/releases/download/rubyinstaller-head/rubyinstaller-head-x64.7z"
+  "head": "https://github.com/MSP-Greg/ruby-loco/releases/download/ruby-master/ruby-mingw.7z",
```

Comment:
I'm a bit hesitant to switch from rubyinstaller-head to ruby-loco here, because all release Windows builds are from RubyInstaller, so it creates some inconsistency.

Maybe we should allow both?
How about having `head` refer to rubyinstaller-head and mingw + mswin refer to ruby-loco builds?

Reply:
Ok.

Reply:
Done.